plugins/async-node/core/src/index.ts

```diff
@@ -166,11 +166,13 @@ export class AsyncNodePluginPlugin implements AsyncNodeViewPlugin {
               options.parseNode && result
                 ? options.parseNode(result)
                 : undefined;
-
             if (parsedNode) {
               this.resolvedMapping.set(node.id, parsedNode);
-              view.updateAsync();
+            } else {
+              // If parsedNode is undefined, update the resolvedMapping with the existing node (supporting null | undefined) as of now
+              this.resolvedMapping.set(node.id, node);
```

Comment:
the existing node is an async node, correct? Does this not trigger additional rounds of the hook calls?

Reply:
Good catch ! How about this .. 
              if (!this.resolvedMapping.has(node.id)) {
                this.resolvedMapping.set(node.id, node);
              } // This is to ensure that the node is not resolved again